test/partnership.e2e-spec.ts

```diff
@@ -157,6 +157,51 @@ describe('Partnership e2e', () => {
     );
   });
 
+  it.only('update mou overrides partnership', async () => {
```

Comment:
Shouldn't this just be apart of the update test?

Reply:
My preference is to keep tests small and just make more tests for each specific permutation.

Reply:
Ok
```suggestion
  it('update mou overrides partnership', async () => {
```

Reply:
forgot to remove that!